lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -144,6 +144,28 @@
   ]
 end
 
+desc 'Run puppet-lint and fix issues automatically'
+PuppetLint::RakeTask.new(:lint_fix) do |config|
+  config.fail_on_warnings = true
+  config.fix = true
+  config.disable_checks = %w[
+    80chars
+    140chars
+    class_inherits_from_params_class
+    class_parameter_defaults
+    documentation
+    single_quote_string_with_variables
+  ]
+  config.ignore_paths = [
+    'bundle/**/*.pp',
+    'pkg/**/*.pp',
+    'spec/**/*.pp',
+    'tests/**/*.pp',
+    'types/**/*.pp',
+    'vendor/**/*.pp',
```

Comment:
on Vox Pupuli, we also ignore `.vendor`. Maybe you want to add that as well:
https://github.com/voxpupuli/modulesync_config/blob/5a7d028a95278d11c2b02616d300fda002fefa0d/moduleroot/Rakefile.erb#L22